tools/eslint-rules/crypto-check.js

```diff
@@ -70,7 +70,16 @@ module.exports = function(context) {
 
   function report(nodes) {
     nodes.forEach((node) => {
-      context.report(node, msg);
+      context.report({
+        node,
+        message: msg,
+        fix: (fixer) => {
+          return fixer.insertTextAfter(
+            node,
+            '\n\nif (!common.hasCrypto)\n\tcommon.skip("missing crypto");'
```

Comment:
What does the `node` refer to in this context? This should be getting inserted right after `common` is required rather than somewhere random.

Also, the project doesn't use tabs but rather spaces so `\t` should be a double space.

Reply:
Thanks for the comments. Working of commit to fix.

Reply:
I've pushed the changes. Also please note that I've added a regExp in rules-utils.js, please verify the validity of the same, or suggest if there are any better ways of doing that.